src/styles/antd-overrides/collapse.scss

```diff
@@ -69,33 +69,37 @@
     border-radius: 2px;
   }
   .ant-collapse-content-box {
-    border: 1px solid var(--stroke-secondary);
-    box-shadow: none !important;
+    border: 1px solid var(--stroke-tertiary);
+    box-shadow: 0px 8px 12px rgba(0, 0, 0, 0.1);
     padding: 0;
     width: 100%;
     background-color: var(--background-l0);
   }
-  .ant-select-arrow {
-    display: none;
-  }
-  .nav-dropdown-item {
-    display: flex;
-    justify-content: flex-start;
-    align-items: center;
-    border: none;
-    padding-left: 10px;
-    width: 100%;
-    height: 40px;
-    color: var(--text-primary);
-    font-weight: 400;
-    cursor: pointer;
+  .ant-collapse-content-box.dark {
+    box-shadow: 0px 8px 12px #000000;
   }
-  .ant-collapse-content-inactive {
+  .ant-select-arrow {
     display: none;
   }
-  .nav-dropdown-item:hover {
-    background-color: var(--background-l1);
-  }
+}
+
+.nav-dropdown-item {
```

Comment:
**question**; does this have to be a class? The best approach is debatable, but inline styles in JSX are more commonly used in the codebase. WDYT?

Reply:
Yep it's a good call. I've moved pretty much all non-antd styles out of the `css` files and into a `navStyles.ts` typescript file